content/en/docs/usage/kubectl-plugin.md

```diff
@@ -13,24 +13,29 @@ While the kubectl plugin is supported, it is recommended to use
 [cmctl](../cmctl/) as this enables a better experience via tab auto-completion.
 
 ## Installation
+
 You need the `kubectl-cert-manager.tar.gz` file for the platform you're using, these can be found on our [GitHub releases page](https://github.com/jetstack/cert-manager/releases).
 In order to use the kubectl plugin you need its binary to be accessible under the name `kubectl-cert_manager` in your `$PATH`.
 
-### macOS/Linux
+### Linux/MacOS
+
 Run the following commands to set up the plugin:
+
 ```console
-$ curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-linux-amd64.tar.gz
-$ tar xzf kubectl-cert-manager.tar.gz
-$ sudo mv kubectl-cert_manager /usr/local/bin
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
```

Comment:
```suggestion
OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-$OS-$ARCH.tar.gz
```

Oops. I missed this in my last review. The download URL needs to point at the `kubectl-cert-manager.tar.gz` archive.

```console
$ OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -L -o kubectl-cert-manager.tar.gz https://github.com/jetstack/cert-manager/releases/latest/download/kubectl-cert_manager-$OS-$ARCH.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   171  100   171    0     0    485      0 --:--:-- --:--:-- --:--:--   485
100   676  100   676    0     0   1258      0 --:--:-- --:--:-- --:--:--  660k
100 14.0M  100 14.0M    0     0  7424k      0  0:00:01  0:00:01 --:--:-- 15.2M

```

```console
$ tar --list --file kubectl-cert-manager.tar.gz
LICENSES
kubectl-cert_manager

```